app/api/helpers/system_mails.py

```diff
@@ -63,7 +63,7 @@
         'message': (
             u"Hi {email},<br/>" +
             u"Thank You for participating in our event. We hope you enjoyed it. "
-            u"Please check the list of more upcoming events. <br />" +
+            u"Please check out other upcoming events around you on eventyay.com <br />" +
```

Comment:
`Take out the list of upcoming events and instead link to eventyay.com` 
@codedsun

Reply:
Please read my comment here 
https://github.com/fossasia/open-event-server/pull/6624#issuecomment-558815447

@kushthedude

Reply:
Remove the list of upcoming events which are being shown .

Reply:
Omg, I thought take out means extraction -_-